__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  test("GET:200 responds with an array of all topic objects with slug and description properties ", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics[0]).toHaveProperty("slug");
+        expect(topics[0]).toHaveProperty("description");
```

Comment:
Rather than just checking the first item in the array, we should be checking each item in it. Can you think of a way we can do this? 🙂